locales/tr-TR.yml

```diff
@@ -0,0 +1,239 @@
+hexpatch: HexPatch
+
+errors:
+  enable_raw_mode: "Ham mod etkinleştirilemedi"
+  setup_commands: "Kurulum komutları çalıştırılamadı"
+  create_terminal: "Terminal oluşturulamadı"
+  clear_terminal: "Terminal temizlenemedi"
+  create_app: "Uygulama oluşturulamadı"
+  disable_raw_mode: "Ham mod devre dışı bırakılamadı"
+  teardown_commands: "Kapatma komutları çalıştırılamadı"
+  show_cursor: "İmleç gösterilemedi"
+  detect_terminal_theme: "Terminal teması algılanamadı: %{e}"
+  load_settings: "Ayarlar yüklenirken hata oluştu: %{e}"
+  connect_ssh: "%{ssh} adresine bağlanılamadı: %{e}"
+  load_plugins: "Eklentiler yüklenirken hata oluştu: %{e}"
+  create_comments: "Yorumlar oluşturulamadı: %{e}"
+  write_comments: "Yorumlar yazılamadı: %{e}"
+  parse_comments: "Yorumlar ayrıştırılamadı: %{e}"
+  read_comments: "Yorumlar okunamadı: %{e}"
+  no_symbols_found: "Sembol bulunamadı"
+  no_matching_symbols: "Arama desenine uyan sembol bulunamadı"
```

Comment:
Maybe "Aramaya" instead of "Arama desenine" because "desen" almost sounds out of place for "pattern" in this context... But a second thought? @orhun